openml/extensions/sklearn/extension.py

```diff
@@ -1531,15 +1534,31 @@ def _prediction_to_probabilities(y: np.ndarray, classes: List[Any]) -> np.ndarra
             -------
```

Comment:
I can't comment on line L#1527 as it is not changed this commit. But I noticed that the parameter name in the docstring is different from the parameter name in the signature. Can you fix this?